### DISCUSSION TITLE
This theme keeps crashing wordpress-what happened?

### DISCUSSION THREAD

**Discussion by rvakid82** [bootscore/bootscore#911] [General]
I've used bootscore for years and years without any issue. I upgraded to v6 and i get an error message "There has been a critical error on this website." with no instructions. What happened? I used ot use this framework to build so many sites and now none of them work. What's wrong? Please help.

**Comment by crftwrk** [comment]
Hi,

it seems that you have a function twice or something is missing. However, in v6 everything has changed and it's not possible to simply update existing v5 to v6 without migration. 

Please read https://bootscore.me/blog/bootscore-6-0-0-beta1/ first to understand what's the different. Then follow each step in the migration guide https://bootscore.me/documentation/migration/#v6-0-0-beta-1 on how to migrate from v5 to v6.

You should start with a new child-theme and then transferring step-by-step changes from your v5 child to the new v6 child.

If you need help here, feel free to ask anytime, but make sure that you have shared a link to your site.

**Comment by rvakid82** [comment]
Thank you for the reply. I was working on a local site so I couldn't share the URL but for some reason it resolved itself after I reinstalled everything? I read the documentation to try to understand what it different but my main takeaway is I need to rebuild the entire theme? Will this be necessary each time this framework is updated? I feel like this used to be a more simple theme to just install in wordpress and customize HTML, CSS, JS and PHP files. This seems much more complicated now? How do I learn about all this?

**Reply by crftwrk** [reply]
> what it different but my main takeaway is I need to rebuild the entire theme?

Yes, v5 to v6 was a huge step and it is a new theme. It is designed to use it with [filters](https://bootscore.me/documentation/filters/) to avoid template overrides and has much more features and improvements. It has an [auto-updater](https://bootscore.me/documentation/theme/updates/) now which makes the usage of a child-theme mandatory or you must remove that if you prefer to work on the main-theme.

> This seems much more complicated now? 

No, it's more simple and future proof. But of course, new and different which makes it at first glance more complicated.

> How do I learn about all this?

To help users migrate from v5 to v6, we have a [migration guide](https://bootscore.me/documentation/migration/), explaining each step to upgrade safely. Check the [blog](https://bootscore.me/blog/) where you will find a description what's new in each release. Follow us on [Twitter](https://x.com/_bootscore) to stay up-to-date, and of course, everything what you need to know is written down in the [docs](https://bootscore.me/documentation/). If you want to know more about upcoming releases, have a look to the [project board](https://github.com/orgs/bootscore/projects).

- https://bootscore.me/blog/bootscore-6-0-0-beta1/
- https://bootscore.me/blog/bootscore-6/
- Check the v6 [changelog](https://github.com/bootscore/bootscore/releases/tag/v6.0.0-beta1)

Does this help?
